### ISSUE TITLE
Unusual Trend in Validation Loss

### ISSUE THREAD

**Issue by SidPad** [microsoft/MoCapAct#15]
Hi,

Thanks for your help in advance.
I tried to reproduce the result for the motion completion, I trained the GPT policy and visualized the action prediction in MuJoCo. I stopped the training early, thinking that I might have made a mistake since the validation losses did not follow the usual trend (following a decreasing trend) and there might have been a possibility of overfitting.
However, the visualization in MuJoCo looked fine, indicating that the model has learnt to predict the actions quite well.
Is there any reason why the validation losses are misleading? Or am I interpreting them wrong?

![MotionCompletionGraph](https://github.com/microsoft/MoCapAct/assets/56720486/4bf478e9-1a2f-44cc-9f7f-9c0c07525925)

Please find the reproduced results attached to this message. The model was trained on the 600 GB dataset and I have used different values of learning rate, batch_size, and validation frequency, but the increasing trend is observed in most.

Thanks again for your help.

**Comment by nolanwagener** [comment]
Hi Sid, my apologies for the very belated response.

I've also seen this validation loss behavior in my training runs. And yes, while the increasing validation loss is concerning on paper, the GPT policy can still competently control the robot on validation clips.

The main takeaway is that supervision performance is not necessarily aligned with downstream control performance. For example, a given prompt may be too ambiguous to figure out the next action precisely. For instance, an agent that has been standing still can either continue to stand still or begin to walk. As long as our policy can do either action competently, then it has good control performance. But the supervision performance on the validation set may not indicate that.
